include/sdf/Param.hh

```diff
@@ -30,7 +30,12 @@
 #include <variant>
 #include <vector>
 
-#include <ignition/math.hh>
+#include <ignition/math/Angle.hh>
+#include <ignition/math/Color.hh>
+#include <ignition/math/Pose3.hh>
+#include <ignition/math/Quaternion.hh>
+#include <ignition/math/Vector2.hh>
+#include <ignition/math/Vector3.hh>
```

Comment:
Even though this is the right thing to do, we should minimize the chances of breaking downstream users on a stable version. That's why we decided to target https://github.com/gazebosim/gz-transport/pull/315 at Garden for example - that specific include change will definitely break lots of users. Here it could be argued that `Param.hh` is not as widely used as `Node.hh`, but I think the same principle applies.